src/Features/Core/Portable/CodeRefactorings/RefactoringHelpersService.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CodeRefactorings
+{
+    abstract class RefactoringHelpersService : IRefactoringHelpersService
```

Comment:
Nit: we always explicitly specify the accessibility modifier, please add `internal`